Bio/SeqIO/QualityIO.py

```diff
@@ -386,6 +388,8 @@
 # Solexa offsets.
 SANGER_SCORE_OFFSET = 33
 SOLEXA_SCORE_OFFSET = 64
+INVALID_CHAR_CODE = 200
```

Comment:
I appreciate there is a lot of flexibility here, but why 200? Why not say 255 at the top of the ASCII range?

Reply:
I need 255, 254, 253, 252, and 251 to be -1, -2, -3, -4, and -5 (for signed integers), so I picked an clean number far away from either side. Not sure whether there is a better choice

Reply:
Ah - that makes sense, I'd forgotten about minus scores when mapping between the old Solexa scheme.

Reply:
I didn’t know that even existed until the tests started failing when I assumed they were positive. I’m glad they got rid of negative scores in the newer systems